build-system/babel-config/helpers.js

```diff
@@ -46,24 +47,20 @@ function getExperimentConstant() {
 function getReplacePlugin() {
   /**
    * @param {string} identifierName the identifier name to replace
-   * @param {boolean} value the value to replace with
+   * @param {boolean|string} value the value to replace with
    * @return {!Object} replacement options used by minify-replace plugin
    */
   function createReplacement(identifierName, value) {
-    return {
-      identifierName,
-      replacement: {type: 'booleanLiteral', value: !!value},
-    };
+    const replacement =
+      typeof value === 'boolean'
+        ? {type: 'booleanLiteral', value}
+        : {type: 'stringLiteral', value};
```

Comment:
For your consideration; typechecking guarantees the possible results from `typeof`
```suggestion
    const replacement = {value, type: `${typeof value}Literal`};
```

Reply:
Very clever! I don't know how I feel about this 🤔